src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+
+For example, users cannot know their balances if an operator withholds details about the rollup’s state. Also, they cannot perform state updates using information contained in a newly added block. 
+
+
+#### Data availability and sharding {data-availability-and-sharding}
+
+Although rollups can scale throughput with off-chain computation, their capacity is limited by the data throughput of the underlying blockchain. If rollups are to use Ethereum as a data availability layer, it must increase its data storage and processing capabilities. 
+
+[Sharding](/upgrades/shard-chains/) is a  proposed method for increasing data throughput on Ethereum’s execution layer. In sharding, the network is split into a select number of sub-chains—each with a dedicated set of validators. 
+
+Validators will only need to run full nodes for their shard and run in light-client capacity for other shards. Sharding increases data space available to rollups since the job of storing data is split across different shards. 
+
+But data sharding introduces a new problem: “What if validators on one shard become malicious and start processing invalid state transitions?”. This is possible since full nodes no longer have access to the same transaction data as is currently the case. 
+
+Implementing data sharding requires creating a system for nodes to verify data availability in other shards without downloading blocks. If validators must download data dumped on every shard, then the purpose of sharding is defeated. 
+
+## Types of data availability systems {#types-of-data-availability-systems} 
+
+### On-chain data availability {#on-chain-data-availability} 
+
+The standard solution to solving data availability is to force block producers to publish all transaction data and have validating nodes download it. Ethereum uses this method, as do rollups (and other scaling solutions) that post transaction data on the L1 blockchain. 
+
+The problem is that on-chain data availability places bottlenecks on scalability. Full nodes have to download the same data and check it, which reduces the number of transactions that can be processed per second.
+
+More importantly, on-chain DA has negative implications for decentralization since it increases the data that full nodes must hold. If Ethereum’s state spirals, validators must invest in more expensive machines—which would probably reduce the number of people willing to run a validating node.  
+
+### Off-chain data availability {#off-chain-data-availability} 
+
+Off-chain data availability solutions move data storage off Ethereum's base layer. The block producer doesn't publish the data on-chain, but provides a cryptographic commitment to prove the availability of the data. 
+
+This is a method used by some scaling solutions, such as [validiums](/developers/docs/scaling/validium/), to reduce the amount of data posted on Ethereum. The drawback is that this system requires trusting block producers to release the data if necessary.  
```

Comment:
We should be stronger in explaining the negative implications of off-chain DA here. 

If I read this as someone who didn't understand DA at all, these would feel more equally weighted in terms of potential solutions to the data availability problem than they actually are.

Reply:
+1 It kind of undermines the "don't trust, verify" maxim that I suggest we repeat at the start of the intro